docs/configuration/relocation/README.md

```diff
@@ -210,6 +210,13 @@ To configure automatic dependency relocation, set `enableAutoRelocation = true`
 > Be mindful that some Gradle plugins will automatically add dependencies to your class path. You may need to remove these
 > dependencies if you do not intend to shadow them into your library.
 
+
+## Relocating Kotlin Standard Library and Reflect
+
+It is not recommended to relocate Kotlin Standard Library if you are using Kotlin Metadata or Kotlin Reflect in the
```

Comment:
Should we add link to Kotlin Metadata and Kotlin Reflect?
 https://kotlinlang.org/docs/metadata-jvm.html, https://kotlinlang.org/docs/reflection.html

Reply:
Sure! Done.